test/infrastructure/docker/internal/docker/kind_manager_test.go

```diff
@@ -38,7 +38,7 @@ func TestCreateNode(t *testing.T) {
 		{
 			ListenAddress: "0.0.0.0",
 			HostPort:      32765,
-			ContainerPort: KubeadmContainerPort,
+			ContainerPort: 6443,
```

Comment:
```suggestion
			ContainerPort: KubeadmContainerPort,
```
Let's use the existing const